app/services/chat_publisher.rb

```diff
@@ -7,7 +7,11 @@ def self.publish_new!(chat_channel, chat_message, staged_id)
     content[:stagedId] = staged_id
     permissions = permissions(chat_channel)
     MessageBus.publish("/chat/#{chat_channel.id}", content.as_json, permissions)
-    MessageBus.publish("/chat/#{chat_channel.id}/new-messages", { message_id: chat_message.id, user_id: chat_message.user_id }, permissions)
+    MessageBus.publish(
+      "/chat/#{chat_channel.id}/new-messages",
+      { message_id: chat_message.id, user_id: chat_message.user_id, username: chat_message.username },
```

Comment:
Had to add the username since the `CurrentUserSerializer` only includes the `ignored_user` usernames and I need to filter this on the client-side